docs/open/QUICKSTART.md

```diff
@@ -70,8 +96,6 @@ module "dcos" {
   num_public_agents  = "1"
 
   dcos_variant = "open"
```

Comment:
here we should place the explicit version. Its not mandatory but it adds better usability for upgrades and so on.